Source/VMware.vSphereDSC/Classes/InventoryBaseDSC.ps1

```diff
@@ -77,19 +77,21 @@ class InventoryBaseDSC : BaseDSC {
 
         $vCenter = $this.Connection
         $rootFolder = Get-View -Server $this.Connection -Id $vCenter.ExtensionData.Content.RootFolder
+        $foundDatacenter = $null
 
         <#
         This is a special case where only the Datacenter name is passed.
         So we check if there is a Datacenter with that name at root folder.
         #>
         if ($this.Datacenter -NotMatch '/') {
             $datacentersFolder = Get-Inventory -Server $this.Connection -Id $rootFolder.MoRef
-            try {
-                return Get-Datacenter -Server $this.Connection -Name $this.Datacenter -Location $datacentersFolder -ErrorAction Stop | Where-Object { $._ParentFolderId -eq $datacentersFolder.Id }
-            }
-            catch {
-                throw "Datacenter with name $($this.Datacenter) was not found at $($datacentersFolder.Name). For more inforamtion: $($_.Exception.Message)"
+            $foundDatacenter = Get-Datacenter -Server $this.Connection -Name $this.Datacenter -Location $datacentersFolder | Where-Object { $_.ParentFolderId -eq $datacentersFolder.Id }
```

Comment:
Why try..catch is removed? Cmdlet will fail with error if datacenter is not found by name